### ISSUE TITLE
RuntimeError: JPMML-SparkML not found on classpath

### ISSUE THREAD

**Issue by MauricioLins** [jpmml/pyspark2pmml#29]
Hi,

I am trying to run the example available on the website to get more understanding of it, but I am getting this error:
```
Traceback (most recent call last):
  File ".../train.py", line 38, in <module>
    fit_convert_model("red")
  File ".../train.py", line 32, in fit_convert_model
    PMMLBuilder(sc, df, pipelineModel) \
  File ".../lib/python3.6/site-packages/pyspark2pmml/__init__.py", line 14, in __init__
    raise RuntimeError("JPMML-SparkML not found on classpath")
RuntimeError: JPMML-SparkML not found on classpath
```
I've tried to execute it in my local machie and from my spark cluster and in both I got the same error.
Frankly, I don't know what's going on.

Has anyone seen this error before?

Thanks.

**Comment by vruusmann** [comment]
> RuntimeError: JPMML-SparkML not found on classpath

Have you tried adding the [JPMML-SparkML](https://github.com/jpmml/jpmml-sparkml) library to your PySpark application classpath as detailed in the README file?

Works perfectly:
```
$ export SPARK_HOME=/opt/spark-3.0.0-preview2/
$ $SPARK_HOME/bin/pyspark --packages org.jpmml:jpmml-sparkml:1.6.0
```

And then copy&paste the "DecisionTreeIris" example code from the README file.

TLDR: RTFM
